docs/src/refman/graphics.txt

```diff
@@ -2040,6 +2040,20 @@ Since: 5.1.2
 
 See also: [al_set_render_state]
 
+
+### API: al_get_render_state
+
+Returns one of several render attributes; see [ALLEGRO_RENDER_STATE]
+for details.
+
+This function will return 0 when there is no current display.
+
+Since: 5.2.11
```

Comment:
Since should be 5.2.10. 5.2.9 is the currently released version, while 5.2.10 is the version on master, which will be released next.

Reply:
✅ done